kernel/Dockerfile

```diff
@@ -34,6 +37,7 @@ ARG DEBUG
 ENV KERNEL_SOURCE=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL_VERSION}.tar.xz
 ENV KERNEL_SHA256_SUMS=https://www.kernel.org/pub/linux/kernel/v4.x/sha256sums.asc
 ENV KERNEL_PGP2_SIGN=https://www.kernel.org/pub/linux/kernel/v4.x/linux-${KERNEL_VERSION}.tar.sign
+ENV KERNEL_DEF_CONF=/linux/arch/x86/configs/x86_64_defconfig
```

Comment:
This is not needed here as the only place where the variable is used is in the section below: https://github.com/linuxkit/linuxkit/pull/2185/files#diff-e605bdf17a402b8724f3ef2ca42b291aR61